### PR TITLE
Preparation for 0.4.0 release

### PR DESCRIPTION
**Pull request by orien**
* Remove a few unrequired files from the gem package
* Set Envato as the author
* Bump version 0.3.0 -> 0.4.0
* Mark changes as released in changelog

### CODE REVIEW

File: event_sourcery-postgres.gemspec [hunk 1]
```diff
@@ -6,15 +6,15 @@ require 'event_sourcery/postgres/version'
 Gem::Specification.new do |spec|
   spec.name          = 'event_sourcery-postgres'
   spec.version       = EventSourcery::Postgres::VERSION
-  # TODO: update authors
-  spec.authors       = ['Steve Hodgkiss']
-  spec.email         = ['steve@hodgkiss.me']
+
+  spec.authors       = ['Envato']
+  spec.email         = ['rubygems@envato.com']
```
Comment (orien) [inline event_sourcery-postgres.gemspec]: Is this the direction we want to move. Would we prefer to name each developer who contributes?

Comment (orien) [inline event_sourcery-postgres.gemspec]: Perhaps we name contributors in a section of the readme in similar fashion to https://github.com/envato/double_entry/pull/131

Comment (grassdog) [inline event_sourcery-postgres.gemspec]: Yep, add a separate contributors section I reckon.

Comment (orien) [inline event_sourcery-postgres.gemspec]: Cool, let's look at that in a separate PR.

**Comment by orien** [comment]
Good question @grassdog. 🤔 I think that's worthy of a GitHub issue and round of discussions.

**Pull request merged**